shell/platform/darwin/ios/framework/Source/FlutterPlatformViewsController.mm

```diff
@@ -830,7 +830,7 @@ - (BOOL)submitFrame:(std::unique_ptr<flutter::SurfaceFrame>)background_frame
       self.layerPool->RemoveUnusedLayers();
   self.layerPool->RecycleLayers();
 
-  auto task = [&,                                                         //
```

Comment:
For future reader - could you document more details about this reland?

Reply:
Good point. Updated the description.